app/models/conference.rb

```diff
@@ -25,6 +25,18 @@ def when
     DateFormatter.new(start_date, end_date).to_s
   end
 
+  def cfp_status
+    if cfp_end_at
+      if cfp_end_at > Time.now
+        'open'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.